src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -147,22 +147,25 @@ def rewrite_jsvm_lcov(self, zip_file_path):
             while not self.build_finished:
                 self.build_finished_cv.wait()
 
-        out_dir = zip_file_path[:-4]
+        out_name = zip_file_path[:-4]
+        out_dir = out_name
+        out_file = out_name + '.info'
 
         zip_file = zipfile.ZipFile(zip_file_path, 'r')
         zip_file.extractall(out_dir)
         zip_file.close()
 
-        lcov_files = [os.path.abspath(os.path.join(out_dir, f)) for f in os.listdir(out_dir)]
-        run_check(['gecko-env', './mach', 'python', 'python/mozbuild/mozbuild/codecoverage/lcov_rewriter.py'] + lcov_files, cwd=self.repo_dir)
+        run_check([
+            'gecko-env', './mach', 'python',
+            'python/mozbuild/mozbuild/codecoverage/lcov_rewriter.py',
+            os.path.abspath(out_dir),
+            '--output-file', os.path.abspath(out_file)
+        ], cwd=self.repo_dir)
 
+        lcov_files = [os.path.join(out_dir, f) for f in os.listdir(out_dir)]
         for lcov_file in lcov_files:
             os.remove(lcov_file)
```

Comment:
I can use shutil.rmtree here and delete the entire directory, instead of implementing it manually.

Reply:
Fixed with ac4dc57e5f409d6c702ba39ad296ce0156cb978a.